src/display/api.js

```diff
@@ -418,6 +419,7 @@ function getDocument(src = {}) {
       useWasm,
       useWorkerFetch,
       cMapUrl,
+      iccUrl,
```

Comment:
This property should have a default entry in the following structure as well.
https://github.com/mozilla/pdf.js/blob/af89e77124bf9ccf271d27e3d3f619ce5878c7c4/src/core/evaluator.js#L84-L99